BrowserKit/Sources/ComponentLibrary/ContextualHintView/ContextualHintView.swift

```diff
@@ -152,6 +153,12 @@ public class ContextualHintView: UIView, ThemeApplicable {
     public func applyTheme(theme: Theme) {
         closeButton.tintColor = theme.colors.textOnDark
         descriptionLabel.textColor = theme.colors.textOnDark
+        // Ecosia: Update custom background theming
+        if theme.type == .light {
+            backgroundColor = UIColor(red: 0.153, green: 0.322, blue: 0.263, alpha: 1)
+        } else {
+            backgroundColor = UIColor(rgb: 0xAFE9B0)
```

Comment:
Don't we have those inside `EcosiaTheme`? And if not, should we add them and use from there?

Reply:
Good point, didn't take into consideration that this was inside `BrowserKit`, we can leave it like that for now then 🙂 

I agree it's good to think of a solution given themes should be accessible everywhere, maybe moving them into the shared package? Anyway, we can brainstorm some ideas